draft-irtf-cfrg-vdaf.md

```diff
@@ -3020,8 +3029,21 @@ removing it after unsharding.
 
 # IANA Considerations
 
-This document makes no request of IANA.
+| Value      | Scheme               | Type | Reference |
+|:-----------|:---------------------|:-----|:----------|
+| 0x00000000 | reserved for testing | n/a  | n/a       |
```

Comment:
I think it'd be a good idea to reserve everything with the prefix `0x000` for testing, just in case someone has a test scenario that requires multiple fake (V)DAFs interacting somehow.

Reply:
We can probably do that. Once concern I have is that 32 bits might be too large. Typically codepoints are just 8-16 bits, in which case we wouldn't be able to afford allocating lots of space.

Reply:
Then reserving `0x000000XX` should suffice.

Reply:
Done